src/client/user-state.js

```diff
@@ -1,19 +1,19 @@
-import * as Cookies from "js-cookie";
+import Cookies from "js-cookie";
 
 window.addEventListener("message", function(event) {
 	// Don't parse anything that isn't ours
 	if (event.origin !== window.location.origin) return;
 
 	// If the authorization state has changed, update the user data
 	if (event.data === userState.AUTH_CHANGED) {
-		userState.data = Cookies.getJSON("user_data");
+		userState.data = JSON.parse(Cookies.get("user_data"));
 	}
 }, false);
 
 let userState = {
 	AUTH_CHANGED: 0,
 
-	data: Cookies.getJSON("user_data")
+	data: JSON.parse(Cookies.get("user_data"))
```

Comment:
![image](https://user-images.githubusercontent.com/30075631/134057219-746d61cb-d11a-434d-8ae5-f5fd69eb985c.png)
When I logged out on the client page and refreshed, the page failed to load fully and got this in my console.